### PR TITLE
pagination, configuration webmail and modifications of stats

### PR DESCRIPTION
**Pull request by CilooW**
added pagination, webmail configuration server.

### CODE REVIEW

**Comment by CilooW** [comment]
@Fradou @janvierm @2binfree

File: app/Resources/views/Back/userstats.html.twig [hunk 1]
```diff
@@ -2,19 +2,32 @@
...
             <thead>
                 <th>Genre</th>
                 <th>Nombre</th>
+                <th>Pourcentage</th>
             </thead>
             {% for gender in genders %}
                 <tr>
                     <td>{{ gender.gender }}</td>
                     <td>{{ gender.nb }}</td>
+                    <td>{{ gender.nb/totalusers*100 }}%</td>
```
Comment (2binfree) [inline app/Resources/views/Back/userstats.html.twig]: {{ (gender.nb / totalUsers) * 100 | round(2, 'floor') }}

File: app/Resources/views/Back/userstats.html.twig [hunk 2]
```diff
@@ -23,60 +36,90 @@
...
             <thead>
                 <th>Statut</th>
                 <th>Nombre</th>
+                <th>Pourcentage</th>
             </thead>
             {% for status in statuses %}
                 <tr>
                     <td>{{ status.status }}</td>
                     <td>{{ status.nb }}</td>
+                    <td>{{ status.nb/totalusers*100 }}%</td>
```
Comment (2binfree) [inline app/Resources/views/Back/userstats.html.twig]: {{ (status.nb / totalUsers) * 100 | round(2, 'floor') }}

File: app/Resources/views/Back/userstats.html.twig [hunk 3]
```diff
@@ -23,60 +36,90 @@
...
             <thead>
             <th>Département</th>
             <th>Nombre</th>
+            <th>Pourcentage</th>
             </thead>
             {% for department in departments %}
                 <tr>
                     <td>{{ department.department }}</td>
                     <td>{{ department.nb }}</td>
+                    <td>{{ department.nb/totalusers*100 }}%</td>
```
Comment (2binfree) [inline app/Resources/views/Back/userstats.html.twig]: {{ (department.nb / totalUsers) * 100 | round(2, 'floor') }}

File: app/Resources/views/Back/userstats.html.twig [hunk 4]
```diff
@@ -23,60 +36,90 @@
...
             <thead>
                 <th>Catégorie d'âge</th>
                 <th>Nombre</th>
+                <th>Pourcentage</th>
             </thead>
             {% for age in agecategory %}
                 <tr>
                     <td>{{ age[0] }}</td>
                     <td>{{ age[1] }}</td>
+                    <td>{{ age[1]/totalusers*100 }}%</td>
```
Comment (2binfree) [inline app/Resources/views/Back/userstats.html.twig]: {{ (age[1] / totalUsers) * 100 | round(2, 'floor') }}

**Pull request merged**